easybuild/easyconfigs/m/Mako/Mako-1.0.7-GCCcore-6.4.0-Python-2.7.14-bare.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'PythonPackage'
+
+name = 'Mako'
+version = '1.0.7'
+versionsuffix = '-Python-%(pyver)s-bare'
+
+homepage = 'http://www.makotemplates.org'
+description = """A super-fast templating language that borrows the best ideas from the existing templating languages"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = [
+    '4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae',  # Mako-1.0.7.tar.gz
+]
+
+builddependencies = [
+    # use same binutils version that was used when building GCCcore
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('Python', '2.7.14', '-bare'),
```

Comment:
Before merging this, I'd like to settle the discussion on https://github.com/easybuilders/easybuild-easyconfigs/pull/5072